plugins/modules/aoscx_loopback_interface.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (C) Copyright 2019-2022 Hewlett Packard Enterprise Development LP.
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "certified",
+}
+
+DOCUMENTATION = """
+---
+module: aoscx_loopback_interface
+version_added: "5.0.0"
+short_description: Create or Delete Loopback Interface configuration on AOS-CX
+description: >
+  This modules provides configuration management of Loopback Interfacess on AOS-CX
+  devices.
+author: Alexis La Goutte (@alagoutte)
+options:
+  loopback_id:
+    description: >
+      The ID of this Loopback interface.
+    required: true
+    type: str
+  ipv4:
+    description: >
+      The IPv4 address and subnet mask in the address/mask format. The first
+      entry in the list is the primary IPv4, the remaining are secondary IPv4.
+      To remove an IP address pass in the list without the IP address yo wish
+      to remove, and use the update state.
+    type: list
+    elements: str
+    required: false
+  vrf:
+    description: >
+      The VRF the Loopback interface will belong to once created. If none provided,
+      the interface will be in the Default VRF. If the Loopback interface is
+      created and the user wants to change the interface loopback's VRF, the user
+      must delete the Loopback interface then recreate the Loopback interface in the
+      desired VRF.
+    type: str
+    required: false
+  description:
+    description: Loopback description
+    required: false
+    type: str
+  state:
+    description: Create or update or delete the Loopback.
+    required: false
+    choices:
+      - create
+      - update
+      - delete
+    default: create
+    type: str
+"""
+
+EXAMPLES = """
+  - name: Create Loopback Interface 100
+    aoscx_loopback_interface:
+      loopback_id: 100
+      description: Loopback100
+      ipv4:
+        - 10.10.20.1/24
+
+  - name: Create Loopback Interface 200 with vrf red
+    aoscx_loopback_interface:
+      loopback_id: 200
+      description: Loopback200
+      ipv4:
+        - 10.20.20.1/24
+      vrf: red
+
+  - name: Delete Loopback Interface 100
+    aoscx_loopback_interface:
+      loopback_id: 100
+      state: delete
+"""
+
+RETURN = r""" # """
+
+from ansible.module_utils.basic import AnsibleModule
+
+try:
+    from pyaoscx.device import Device
+
+    USE_PYAOSCX_SDK = True
+except ImportError:
+    USE_PYAOSCX_SDK = False
```

Comment:
@tchiapuziowong do you plan to remove old "format" (without PYAOSCX_SDK ?)

Reply:
Any new module we produce will only support using PYAOSCX sdk